content/enterprise_influxdb/v1.9/administration/cluster-commands.md

```diff
@@ -664,6 +665,107 @@ Successfully left cluster
   * Removed meta node cluster-meta-node-03:8091 from cluster
 ```
 
+### `node-labels`
+
+Assign user-defined labels to meta and data nodes in a cluster.
+Labels are JSON key-value pairs.
+
+#### Syntax
+
+```bash
+influxd-ctl [options] node-labels <command> [options] [<args>]
```

Comment:
Do we document the node-labels [options] somewhere? 
Didn't see `-nodeid` or `-labels` other than in examples...do we have other options?

Reply:
Looks like these are the only two.